components/automate-gateway/api/iam/v2/common/policy.proto

```diff
@@ -24,42 +31,53 @@ message Statement {
         ALLOW = 0;
         DENY = 1;
     }
+    // Whether the statement allows or denies.
```

Comment:
Above, you designate what the options are for Type. I think that makes sense here, too.